packages/core/test/commands.js

```diff
@@ -3,71 +3,75 @@ const commands = require("../lib/commands");
 const commander = new Command(commands);
 const assert = require("assert");
 
-describe("Commander", function() {
-  before("assert preconditions", function() {
+describe("Commander", function () {
+  before("assert preconditions", function () {
     // These commands are expected to exist in tests.
     assert.notEqual(commands.migrate, null);
     assert.notEqual(commands.compile, null);
     assert.notEqual(commands.console, null);
   });
 
-  it("will infer commands based on initial letters entered", function() {
+  it("will infer commands based on initial letters entered", function () {
     var actualCommand = commander.getCommand("m").command;
     assert.equal(actualCommand, commands.migrate);
   });
 
-  it("will infer commands based on initial letters entered, even when typos exist later", function() {
+  it("will infer commands based on initial letters entered, even when typos exist later", function () {
     var actualCommand = commander.getCommand("complie").command;
     assert.equal(actualCommand, commands.compile);
   });
 
-  it("will NOT infer a command if not given enough information", function() {
+  it("will NOT infer a command if not given enough information", function () {
     // Note: "co" matches "console" and "compile"
     var actualCommand = commander.getCommand("co");
     assert.equal(actualCommand, null);
   });
 
-  it("will infer commands based on initial letters entered, given matches for shorter substrings", function() {
+  it("will infer commands based on initial letters entered, given matches for shorter substrings", function () {
     // Note: "co" matches "console" and "compile"
     var actualCommand = commander.getCommand("com").command;
     assert.equal(actualCommand, commands.compile);
   });
 
-  it("will ignore inferring if full command is specified", function() {
+  it("will ignore inferring if full command is specified", function () {
     var actualCommand = commander.getCommand("console").command;
     assert.equal(actualCommand, commands.console);
   });
 
-  it("will warn and display error for unsupported flags in commands", function() {
+  it("will warn and display error for unsupported flags in commands", async function () {
```

Comment:
I'm not going to hold this back for this but we should make a rule of never using these helper words like "will" or "should" in the test descriptions. We should just write them in the form `it("<verb>s...", () =>...`. That way we don't see
```
it("will ....
it("will ...
```

Reply:
Agree, the verbs makes reading summaries better